CanvasCore/CanvasCore/Annotations/Canvadocs/CanvadocsAnnotationProvider.swift

```diff
@@ -195,10 +195,10 @@ class CanvadocsAnnotationProvider: PSPDFContainerAnnotationProvider {
 }
 
 // Needed for setRotationOffset(_:forPageAt:)
-extension CanvadocsAnnotationProvider: PSPDFAnnotationProviderRefreshing {
-    func prepareForRefresh() {
-    }
-
-    func refreshAnnotationsForPages(at pageIndexes: IndexSet) {
-    }
-}
+//extension CanvadocsAnnotationProvider: PSPDFAnnotationProviderRefreshing {
+//    override func prepareForRefresh() {
+//    }
+//
+//    override func refreshAnnotationsForPages(at pageIndexes: IndexSet) {
+//    }
+//}
```

Comment:
oh good, can we remove these now? Not sure how to test the rotation stuff though.